mdsuite/graph_modules/molecular_graph.py

```diff
@@ -47,8 +52,10 @@ def __init__(self, experiment: object, from_smiles: bool = False, from_configura
         self.smiles_string = smiles_string
         self.species = species
 
-        self.database = Database(name=os.path.join(self.experiment.database_path, "database.hdf5"),
-                                 architecture='simulation')
+        self.database = Database(
+            name=os.path.join(self.experiment.database_path,
+                              "database.hdf5"),
+            architecture='simulation')
```

Comment:
more debatable if this is for the good or worse

Reply:
moved the second line up.